### DISCUSSION TITLE
Help a n00b

### DISCUSSION THREAD

**Discussion by j-ktz** [meisnate12/Plex-Meta-Manager#33] [Q&A] [answered]
So, I'm trying to get this up and running but new here and struggling. I've been following the guides and Wiki but still kinda lost. Here's what I've done:

- moved the script to my NAS where I have python installed
- opened up config.yml.template and put in my info (do I leave .template as the extension or change to .yml?)
- SSHed into my NAS to run the script and get the error "line 1, in <module>
    import argparse, logging, os, re, schedule, sys, time, traceback, datetime
ModuleNotFoundError: No module named 'schedule'"
- am I missing a step?

**Comment by meisnate12** [comment, marked as answer]
>    opened up config.yml.template and put in my info (do I leave .template as the extension or change to .yml?)

delete .template it should be .yml

>    SSHed into my NAS to run the script and get the error "line 1, in
>    import argparse, logging, os, re, schedule, sys, time, traceback, datetime
>    ModuleNotFoundError: No module named 'schedule'"
>    am I missing a step?

you have to install the requirements.txt. I personally had trouble doing this on my QNAP NAS. I either just run the script from my computer or use the docker on the NAS. You don't have to run this from your NAS server

**Reply by j-ktz** [reply]
I definitely have requirements.txt. It's in the asme folder. Did you set up your Docker manually?

**Reply by j-ktz** [reply]
So now I'm trying to install with Docker, pulled the image, launched, but not sure what to fill out in General, Volume, Port, Links, Environment, etc?

**Reply by meisnate12** [reply]
you have to use this command to install the requirments.txt
```shell
pip install -r requirements.txt
```
seen on the [local installation](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Local-Installation) page. 

I have set up docker manually but I just use container station on QNAP to load my docker containers

**Reply by meisnate12** [reply]
what kind of NAS do you have?

**Reply by meisnate12** [reply]
also, you shouldn't have to change much of the default docker settings except Volume. You want to mount a path to where you plan to store your config files to `/config`

**Reply by j-ktz** [reply]
I have a Synology NAS. That's the only thing I've changed, however when I run the container the config folder is empty?
<img width="640" alt="screenshot 2021-02-15 at 8 03 55 AM" src="https://user-images.githubusercontent.com/59346628/107950226-6d886480-6f64-11eb-9c3c-c2bc11c8df09.png">

**Reply by meisnate12** [reply]
Did you make a config.yml and put it in there?

**Reply by j-ktz** [reply]
I have updated a config.yml.template. I'm assuming remove "template" from the filename? Should that yml file be the only file in the config folder?

**Reply by meisnate12** [reply]
yes remove the .template part and no you will also need a yml file per library. That file is where you configure your collections

**Reply by j-ktz** [reply]
Running the container created the Config folder but it's just empty. I can drag the Config.yml from the zip I download from GitHub but shouldn't there be other things in here?

<img width="627" alt="screenshot 2021-02-15 at 10 03 29 AM" src="https://user-images.githubusercontent.com/59346628/107962546-163ec000-6f75-11eb-891a-f383ff3e900f.png">

**Reply by j-ktz** [reply]
Do you have Discord? Hate to bother but really trying to get this working

**Reply by meisnate12** [reply]
Sohjiro#2008 hit me up there

**Reply by j-ktz** [reply]
Added you. Thx!
